### PR TITLE
Ocrvs 3595 -As a System Admin I want to enable and disable a client integration

### PR DESCRIPTION
**Pull request by CornelMaze**
- As a System Admin I want to enable and disable a client integration
- Changed user-mgnt routes for deactivateSystemClient and reactivateSystemClient to find the System in database by client_id instead of _id.
- Create a GraphQL gateway endpoints in packages/gateway/src/features/integrations which would call user-mgnt deactivateSystemClient and reactivateSystemClient
- Add Enable / Disable (depending on current status) to 3 dot menu in Integrations. Clicking the button will open a Confirm-modal. Clicking confirm will call the GraphQL endpoint made in the previous task

### CODE REVIEW

File: packages/gateway/src/features/integrations/schema.graphql [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+input ClientPayload {
+  client_id: String
+}
+
+type IntegrationResponse {
+  message: String
+}
+
+type Mutation {
+  reactivateSystemClient(clientDetails: ClientPayload): IntegrationResponse
```
Comment (Zangetsu101) [inline packages/gateway/src/features/integrations/schema.graphql:29]: @naftis Would it be better if we return the updated `System` from the mutations?

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]

File: packages/client/src/offline/actions.ts [hunk 2]
```diff
@@ -92,10 +92,10 @@ export type ApplicationConfigUpdatedAction = {
   type: typeof UPDATE_OFFLINE_CONFIG
   payload: { config: IApplicationConfig }
 }
-export const UPDATE_OFFLINE_INTEGRATIONS =
+export const UpdateOfflineIntegrationsAction =
```
Comment (Zangetsu101) [inline packages/client/src/offline/actions.ts:95]: @CornelMaze There might have been a misunderstanding. I would have liked `ApplicationConfigUpdatedIntegrations` to be changed to `UpdateOfflineIntegrationsAction` and not this one.

Comment (CornelMaze) [reply packages/client/src/offline/actions.ts:95]: ok

Comment (CornelMaze) [reply packages/client/src/offline/actions.ts:95]: Alright

**Review by Zangetsu101 — APPROVED** [review]

**Pull request merged**